lumen/sources/intake.py

```diff
@@ -10,38 +10,59 @@ class IntakeSource(Source):
     An IntakeSource loads data from an Intake catalog.
     """
 
-    catalog = param.String(doc="""
-        The name of the catalog; must be provided.""")
-
     dask = param.Boolean(default=False, doc="""
         Whether to return a dask DataFrame.""")
 
-    filename = param.String(doc="Filename of the catalog file.")
+    filename = param.String(doc="Deprecated alias for the URI")
```

Comment:
do you mean  remove this parameter
and the check in the __init__ above (line 26-29)
 ?
if yes , I am totally agree

Reply:
Thanks for replying so quickly. Didn't want to break your setup but it's early days for the library so would like to keep things clean.